internal/services/netapp/netapp_volume_resource_test.go

```diff
@@ -135,6 +135,21 @@ func TestAccNetAppVolume_standardNetworkFeature(t *testing.T) {
 	})
 }
 
+func TestAccNetAppVolume_snapshot(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_netapp_volume", "test")
+	r := NetAppVolumeResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.snapshot(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("create_from_snapshot_resource_id"),
```

Comment:
`create_from_snapshot_resource_id` is only used during creation and is not meant to be persisted after creation. It's similar to other "creation-only" properties that are only relevant during resource creation.